numpy/version.py

```diff
@@ -5,6 +5,7 @@
 vinfo = get_versions()
 version: str = vinfo["version"]
 full_version: str = vinfo['version']
+short_version: str = vinfo['version']
 git_revision: str = vinfo['full-revisionid']
 release = 'dev0' not in version and '+' not in version
```

Comment:
Could you also change `release` to `isrelease`?

Reply:
`isrelease` did not exist previously (1.19). Do you want both to exist ?

Reply:
it neither exists in the master branch !?

Reply:
You are right, NVM.